pkg/webhook/resources/setting/validator.go

```diff
@@ -335,3 +347,12 @@ func getSystemCerts() *x509.CertPool {
 	}
 	return certs
 }
+
+func hasVMBackupInCreatingOrDeletingProgress(vmBackups []*v1beta1.VirtualMachineBackup) bool {
+	for _, vmBackup := range vmBackups {
+		if vmBackup.DeletionTimestamp != nil || vmBackup.Status == nil || !*vmBackup.Status.ReadyToUse {
+			return true
+		}
+	}
+	return false
```

Comment:
Question:
when will   `vmBackup.Status == nil` ?

`if vmBackup.DeletionTimestamp != nil || vmBackup.Status == nil || !*vmBackup.Status.ReadyToUse`
or
`if vmBackup.DeletionTimestamp != nil || (vmBackup.Status != nil && !*vmBackup.Status.ReadyToUse)`

Reply:
There is no `status` in VM Backup when it's just created. If a VM Backup doesn't have `status`, it means it is in creating process.
https://github.com/harvester/harvester/blob/655c251ada5808399c8b9c475abc75e437a51d09/pkg/controller/master/backup/backup.go#L133-L149